### PR TITLE
refactor lib/FeatureServices, standard 5 tweaks

### PR DESCRIPTION
**Pull request by ngoldman**
This PR locks standard to `^5` and refactors lib/FeatureServices in a similar way to #244.

### CODE REVIEW

File: lib/Exporter.js [hunk 1]
```diff
@@ -339,50 +337,47 @@ function callOgr (params, geojson, options, callback) {
   var outFile = params.outFile
   var metadata = options.metadata
 
-  if (fs.existsSync(outFile)) {
-    // we already have the file, just return it
-    callback(null, outFile)
+  // we already have the file, just return it
+  if (fs.existsSync(outFile)) return callback(null, outFile)
```
Comment (ngoldman) [inline lib/Exporter.js]: Also simplified a function in `lib/Exporter` to return earlier and remove an extra layer of indentation.

**Comment by dmfenton** [comment]
It's tough to big refactors like this...is this just about Standard and the one function simplification?

**Comment by ngoldman** [comment]
Sorry, I will try to make PRs with refactoring have diffs that are easier to scan. Trying to at least limit it to one file at a time for now.

Major changes in terms of `lib/FeatureServices` are same as #244:

> * no more use of this -- functions call each other directly
> * functions attached to module exports at the bottom of the file

Tests are passing, both Query and FeatureServices are just used by BaseController and nothing else. I did this conversion in `koop-provider` first, which we've been using for `koop-geotrigger` without issues so far.

**Comment by dmfenton** [comment]
Eh that wasn't really a criticism. I don't think you can do much more than be really explicit in the description about what a reviewer should focus on.

**Comment by ngoldman** [comment]
Alright I added descriptions in all the `TODO: missing description` spots in `lib/FeatureServices`. These are best guesses after tracking down where the functions are used.

**Comment by dmfenton** [comment]
Thanks for doing the dirty work

**Pull request merged**